boards/arm/wio_lte_cat_1/wio_lte_cat_1.dts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2020, Kieran Bingham <kbingham@kernel.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f405Xg.dtsi>
+
+/ {
+	model = "Seeed Wio LTE Cat.1 board";
+	compatible = "seeed,wio-lte-cat1", "st,stm32f405";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,ccm = &ccm0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		rgb_led_pwr: ws2812b_pwr {
+			gpios = <&gpioa 8 GPIO_INT_ACTIVE_HIGH>;
```

Comment:
Please rebase on top of latest master and use new GPIO API

Reply:
Ack. I'll have to see what changed there.